lib/impure/db_postgres.nim

```diff
@@ -519,7 +519,13 @@ proc open*(connection, user, password, database: string): DbConn {.
   ##
   ## Note that the connection parameter is not used but exists to maintain
```

Comment:
Update the documentation too please.

Reply:
I've removed the note about the unused param.